daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -2094,6 +2107,65 @@ private[lf] object SBuiltin {
 
   }
 
+  /** $lookupDisclosedCachedContract :: DisclosedContract -> CachedContract
```

Comment:
New SBuiltin's to support adding disclosed contracts to the on ledger contract cache map